packages/synapse-react-client/package.json

```diff
@@ -227,15 +230,17 @@
     "util": "^0.12.5",
     "vite": "^5.4.3",
     "vite-config": "workspace:*",
+    "vitest": "^1.6.0",
     "weak-napi": "^2.0.2",
     "whatwg-fetch": "^3.6.20"
   },
   "scripts": {
     "clean": "rimraf dist storybook-static coverage *.tsbuildinfo",
     "prepublishOnly": "pnpm install && pnpm nx run synapse-react-client:build",
     "start": "storybook dev -p 6060",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
+    "test": "pnpm run \"/^test:.*/\"",
+    "test:vitest": "vitest",
+    "test:jest": "jest",
```

Comment:
Running both Vitest and Jest separately may bust our total coverage metrics, but it lets us gradually migrate